resources/monitoring/charts/alert-rules/templates/kyma-rules.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       expr: sum(kube_pod_container_status_running { namespace=~"kyma-.*|kube-.*|istio-.*|natss", pod!~"(test.*)|((dummy|sample)-.*)|(.*(docs|backup|test)-.*)|((oct-tp-testsuite-all)-.*)|(.*-(tests|dummy))" } == 0 )by (pod,namespace) * on(pod, namespace) (kube_pod_status_phase{phase="Succeeded"} != 1)
       for: 60s
       labels:
-        severity: critical
+      #  severity: critical
```

Comment:
Seems to be invalid, see build logs: spec.groups.rules.labels in body must be of type object: "null"

Either change the value to something like "warning" or comment out labels section fully

Reply:
Thanks for the tip!